youtube_dl/extractor/mitele.py

```diff
@@ -91,3 +87,37 @@ def _real_extract(self, url):
             'age_limit': int_or_none(info.get('rating')),
             'timestamp': parse_iso8601(pre_player.get('publishedTime')),
         }
+
+        if video.get('dataCmsId') == 'ooyala':
+            info.update({
+                '_type': 'url_transparent',
+                # for some reason only HLS is supported
+                'url': smuggle_url('ooyala:' + video_id, {'supportedformats': 'm3u8,dash'}),
+            })
+        else:
+            config = self._download_json(
+                video['dataConfig'], video_id, 'Downloading config JSON')
+            services = config['services']
+            gbx = self._download_json(
+                services['gbx'], video_id, 'Downloading gbx JSON')
+            caronte = self._download_json(
+                services['caronte'], video_id, 'Downloading caronte JSON')
+            cerbero_data = {
+                'bbx': caronte['bbx'],
+                'gbx': gbx['gbx']
+            }
+            headers = {
+                'Content-Type': 'application/json;charset=UTF-8',
+                'Origin': 'https://www.mitele.es'
+            }
+            cerbero = self._download_json(
+                caronte['cerbero'], video_id, 'Downloading cerbero JSON',
+                headers=headers,
+                data=json.dumps(cerbero_data).encode('utf-8'))
+            stream = '%s?%s' % (caronte['dls'][0]['stream'],
+                                cerbero['tokens']['1']['cdn'])
```

Comment:
Do not encode query into URL but pass via appropriate `_extract_m3u8_formats` parameter.

Reply:
I've passed it through query parameter, but the result isn't very readable from my point of view. Anyway, if this is the way to go, just go ahead and if you expected something different just tell me.